pkg/svc/workflow.go

```diff
@@ -76,7 +77,8 @@ func (s *workflowsrvc) Assign(ctx context.Context, w *workflow.AssignPayload) (e
 		return err
 	}
 
-	url, err := s.backend.CreateListener(ctx, s.logger, w.Name, true)
+	waitTimeout := 1 * time.Minute
```

Comment:
Could you put this somewhere under the global constants?

Reply:
sure I can, however what would be the reason for doing so?

Reply:
Clean code, I guess? Also, better chance to modify it if we want to configure it later...

Reply:
I would agree with you if this variable was also being used somewhere else.

IMO, moving it to a global constant might cause confusion in case there is a timeout in another part of the code. Keeping it close to where it is used makes it easier to understand what to expect in case of changing its value.

Reply:
done.

Reply:
thank you !